src/main/js/index.tsx

```diff
@@ -80,5 +81,9 @@ binder.bind("repos.branch-details.information", ({ repository, branch }) => (
   <CreatePullRequestButton repository={repository} branch={branch} />
 ));
 
+binder.bind("repository.card.quickLink", ({ repository, repositoryLink }) => (
+  <RepositoryPullRequestCardLink repository={repository} repositoryLink={repositoryLink} />
+));
+
```

Comment:
I think this is the same as `binder.bind("repository.card.quickLink", RepositoryPullRequestCardLink)`.

Reply:
I would use a precondition to avoid rendering the extension, if no pr link is available.

Reply:
The guard is inside RepositoryPullRequestCardLink. Isn't this the same?

Reply:
No, it is not. With a guard inside the RepositoryPullRequestCardLink, the component it self is rendered. With a precondition the RepositoryPullRequestCardLink is not rendered at all.